sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -315,6 +316,8 @@ public int getCurrentVolume() {
   @Override
   public void setFullscreen(boolean isFullscreen) {
     _layout.setFullscreen(isFullscreen);
+    setChanged();
+    notifyObservers();
```

Comment:
can you notify Observers with an OoyalaNotificationObject? see the OoyalaPlayer side:

`
  void sendNotification(String notificationName, Object data) {
    sendNotification(new OoyalaNotification(notificationName, data));
  }

  void sendNotification(OoyalaNotification notification) {
    setChanged();
    notifyObservers(notification);
  }
`